admin_manual/configuration/files/external_storage/onedrive.rst

```diff
@@ -0,0 +1,84 @@
+==============================================
+How to Create and Configure Microsoft OneDrive
+==============================================
+
+To use Microsoft OneDrive as an external storage option in ownCloud, you need to do two things:
+
+#. `Create an application configuration`_
+#. `Configure a mount point in ownCloud`_
+
+Create an Application Configuration
+-----------------------------------
+
+.. figure:: ../../../images/configuration/files/external_storage/onedrive/register-an-application.png
+   :alt: Register a new OneDrive application.
+
+To create a new application: 
+
+- Open https://apps.dev.microsoft.com/ in your browser of choice and click "*Create App*".
+- Under "*Properties*", set the application's name. 
```

Comment:
Please add where the application´s name is used in ownCloud or for what it is good.